### DISCUSSION TITLE
Make a NpmJs package

### DISCUSSION THREAD

**Discussion by jalallinux** [abbasudo/laravel-purity#87] [Ideas]
I had an idea to create an npm package for this Laravel package, allowing frontend developers who use this API model to easily meet their needs with this package.

I finally managed to implement this package and have published it publicly on my GitHub and npm.
- [npmjs](https://www.npmjs.com/package/purity-client-js)
- [github](https://github.com/hidevs/purity-client-js)

If possible, please consider donating.
Your donations are greatly appreciated and needed.

- [Fiat Donation](https://reymit.ir/jalallinux)
- BTC wallet: bc1qj4yk9ck7lcdsatmujds2ufea0q56kalhlqregf

I hope this proves useful for my fellow developers.

**Comment by num8er** [comment]
> I had an idea to create an npm package for this Laravel package, allowing frontend developers who use this API model to easily meet their needs with this package.
> 
> I finally managed to implement this package and have published it publicly on my GitHub and npm.
> - [npmjs](https://www.npmjs.com/package/purity-client-js)
> - [github](https://github.com/hidevs/purity-client-js)
> 
> If possible, please consider donating.
> Your donations are greatly appreciated and needed.
> 
> - [Fiat Donation](https://reymit.ir/jalallinux)
> - BTC wallet: bc1qj4yk9ck7lcdsatmujds2ufea0q56kalhlqregf
> 
> I hope this proves useful for my fellow developers.

Looks great 👍 
Nice work!

**Reply by jalallinux** [reply]
That is so sweet. Thanks.
It was a service to the open-source community.

**Comment by abbasudo** [comment]
Hello @jalallinux 
Nicely Done. We appreciate your work. I have included a client package link in [Purity home page](https://abbasudo.github.io/laravel-purity). Wish you the best.
